tests/rws_tests.py

```diff
@@ -32,7 +32,7 @@ def test_no_primary(self):
                 }
             ]
         }
-        fp = FpsCheck(fps_sites=json_dict,
+        fp = RwsCheck(rws_sites=json_dict,
```

Comment:
Not totally sure what `fp` was short for, but should it be updated? Same comment for the other test cases.

Reply:
Fixed the name and tried to make it more descriptive.